### PR TITLE
[ticket/14029] Add mcp_forum_display_actions_* template events

### PR DESCRIPTION
**Pull request by adri1b**
PHPBB3-14029
PHPBB3-14030

### CODE REVIEW

**Comment by adri1b** [comment]
!set Event

**Comment by marc1706** [comment]
Needs to be rebased onto current 3.1.x

File: phpBB/styles/subsilver2/template/mcp_forum.html [hunk 1]
```diff
@@ -57,24 +57,32 @@
...
-			<!-- IF S_CAN_MAKE_NORMAL --><option value="make_normal">{L_MAKE_NORMAL}</option><!-- ENDIF -->
-			<!-- IF S_CAN_MAKE_STICKY --><option value="make_sticky">{L_MAKE_STICKY}</option><!-- ENDIF -->
-			<!-- IF S_CAN_MAKE_ANNOUNCE -->
-				<option value="make_announce">{L_MAKE_ANNOUNCE}</option>
-				<option value="make_global">{L_MAKE_GLOBAL}</option>
-			<!-- ENDIF -->
-		</select>
-		<input class="btnmain" type="submit" value="{L_SUBMIT}" />
+	<td class="cat" colspan="6">
+		<table width="100%" cellspacing="0">
```
Comment (marc1706) [inline phpBB/styles/subsilver2/template/mcp_forum.html]: Why did you add a table here? You shouldn't modify the markup just because it might be more convenient for your expected usage.

**Comment by marc1706** [comment]
Pull request will be deleted upon next release as it seems to have been abandoned.

**Comment by marc1706** [comment]
Closed as previously mentioned.
